client/src/components/pages/Rating.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+const propTypes = {
+  rating: PropTypes.number
+}
+class Rating extends Component {
```

Comment:
Good work trying to write your own component for ratings, but there seems to be an issue as the stars don't clear out after review has been submitted, you can try to get the component to re-render after review has been submitted or check out the react package REACT STARS which can be found here https://www.npmjs.com/package/react-stars

Reply:
Thanks @Daramola98 for the feedback. Would get back to you on this.